dfw-init.php

```diff
@@ -248,7 +251,8 @@ private function targeting() {
 			$targeting['Page'][] = 'search';
 		}
 
-		if ( is_single() ) {
+		// change from is-single() to !(is-home() or is-front-page()
```

Comment:
What's the reasoning behind this change?

Reply:
Response:

> many wordpress themes default a specific page to be a homepage. This
condition excludes home-page and front-page targeting, to refine the
results.

I think this has since been resolved by https://github.com/INN/doubleclick-for-wp/pull/72 and https://github.com/INN/doubleclick-for-wp/pull/74 for the issue you opened, https://github.com/INN/doubleclick-for-wp/issues/61